server/plugin/plugin.go

```diff
@@ -119,3 +127,17 @@ func isValidTaskLink(msg string) ([]string, bool) {
 	}
 	return data, true
 }
+
+// Function to validate the pull request link.
+func isValidPullRequestLink(msg string) ([]string, string, bool) {
+	pullRequestRegex := regexp.MustCompile(constants.PullRequestRegex)
+	link := pullRequestRegex.FindString(msg)
+	data := strings.Split(link, "/")
```

Comment:
this is common logic used in the above function as well. make a function for this, if possible